src/Assignment-3/assignment3.css

```diff
@@ -1,7 +1,60 @@
 /* Our CSS file. */
 
 
-.container, .content {
+.container{
 width: 100%;
 height: 100%;
 }
+.button {
+    border: none;
+    color: white;
+    padding: 15px 30px;
+    text-decoration: none;
+    cursor: pointer;
+    text-align: center;
+    /* border-radius: 25px; */
+ }
+ .button1{
+   background-color: lightgreen;
+  
+   
+ }
+ 
+ .section1{
+   background-color:#F2F2F2; 
+   padding: 20px 20px;
+   margin: 30px 30px;
+ 
+ }
+ .section2{
+   background-color: #F2F2F2; 
+   padding: 20px 20px;
+   margin: 30px 30px;
+  
+ }
+ .content-section1{
+   display: inline-block;
+   width: 50%;
+   
+   
+ }
+ .iframe-section1{
+    float: right;
```

Comment:
wrong formatting it should be inside a container containing 50% with of the top section. And using px values will make it unresponsive.